tests/components/Dropdown.test.tsx

```diff
@@ -184,6 +184,40 @@ describe('Dropdown', () => {
     expect(mockedOnSelectFn).toHaveBeenCalledWith('item1', 0, undefined);
   });
 
+  it('selects when an option is focused on toggle', () => {
+    const mockedOnToggleFn = jest.fn();
+    const dropdownProps: DropdownProps = {
+      screenReaderText: 'screen reader text here',
+      onToggle: mockedOnToggleFn
+    };
+    render(
+      <div data-testid='container'>
```

Comment:
why is a `data-testid` attribute being added here? is it to select the dropdown container or the document?

Reply:
Oh I copied part of a previous test and forgot to remove it, I'll fix that